tests/unit/s2n_tls13_handshake_state_machine_test.c

```diff
@@ -122,6 +171,13 @@ int main(int argc, char **argv)
     /* Test: A WITH_EARLY_DATA form of every non-full, non-retry handshake exists
      *       and matches the non-WITH_EARLY_DATA form EXCEPT for the END_OF_EARLY_DATA
      *       message and client CCS messages.
+     *= https://tools.ietf.org/rfc/rfc8446#5
```

Comment:
new lines between regular comments and compliance comments might make some of this more readable